network-api/networkapi/wagtailpages/templatetags/homepage_tags.py

```diff
@@ -32,17 +32,16 @@ def homepage_image_with_class(context, path, classname):
 
 
 @register.simple_tag
-def get_blog_page_authors(blog_page):
+def get_page_authors(parent_page):
```

Comment:
let's not call this "parent_page" though, this is still just `page`
```suggestion
def get_page_authors(page):
```